verticaltabs.js

```diff
@@ -459,7 +457,17 @@ VerticalTabs.prototype = {
           tabsPopup.hidePopup();
         }
       }
-    });
+    };
+
+    leftbox.addEventListener('mouseenter', enter);
+    leftbox.addEventListener('mousemove', enter);
+    leftbox.addEventListener('mouseleave', exit);
+    let tab_context_menu = document.getElementById('tabContextMenu');
+    let toolbar_context_menu = document.getElementById('toolbar-context-menu');
```

Comment:
Can we pull these two `let` statements up above the `addEventListener`s?  I think it'll read cleaner to keep all the event listener additions together.
Also, why aren't we adding a `mouseenter` event listener to the context menus?